opencti-platform/opencti-front/src/private/components/settings/organizations/SettingsOrganization.tsx

```diff
@@ -0,0 +1,234 @@
+import { graphql, useFragment } from 'react-relay';
+import makeStyles from '@mui/styles/makeStyles';
+import Typography from '@mui/material/Typography';
+import React from 'react';
+import Grid from '@mui/material/Grid';
+import Paper from '@mui/material/Paper';
+import List from '@mui/material/List';
+import ListItem from '@mui/material/ListItem';
+import { Link } from 'react-router-dom';
+import ListItemIcon from '@mui/material/ListItemIcon';
+import { AccountBalanceOutlined } from '@mui/icons-material';
+import ListItemText from '@mui/material/ListItemText';
+import { Theme } from '../../../../components/Theme';
+import { SettingsOrganization_organization$key } from './__generated__/SettingsOrganization_organization.graphql';
+import AccessesMenu from '../AccessesMenu';
+import SettingsOrganizationDetails from './SettingsOrganizationDetails';
+import UserLineTitles from '../users/UserLineTitles';
+import { UserLine } from '../users/UserLine';
+import { useFormatter } from '../../../../components/i18n';
+import FieldOrEmpty from '../../../../components/FieldOrEmpty';
+
+const useStyles = makeStyles<Theme>(() => ({
+  container: {
+    margin: 0,
+    padding: '0 200px 0 0',
+  },
+  gridContainer: {
+    marginBottom: 20,
+  },
+  title: {
+    float: 'left',
+  },
+  paper: {
+    height: '100%',
+    minHeight: '100%',
+    margin: '10px 0 0 0',
+    padding: '15px',
+    borderRadius: 6,
+  },
+}));
+const settingsOrganizationFragment = graphql`
+  fragment SettingsOrganization_organization on Organization {
+    id
+    standard_id
+    name
+    members {
+      edges {
+        node {
+          id
+          user_email
+          name
+          firstname
+          lastname
+          external
+          otp_activated
+          created_at
+        }
+      }
+    }
+    subOrganizations {
+      edges {
+        node {
+          id
+          standard_id
+          name
+        }
+      }
+    }
+    parentOrganizations {
+      edges {
+        node {
+          id
+          standard_id
+          name
+        }
+      }
+    }
+    ...SettingsOrganizationDetails_organization
+  }
+`;
+const SettingsOrganization = ({ organizationData }: { organizationData: SettingsOrganization_organization$key }) => {
+  const classes = useStyles();
+  const { t } = useFormatter();
+  const organization = useFragment<SettingsOrganization_organization$key>(settingsOrganizationFragment, organizationData);
+  const members = organization.members?.edges ?? [];
+  const subOrganizations = organization.subOrganizations?.edges ?? [];
+  const parentOrganizations = organization.parentOrganizations?.edges ?? [];
+
+  const userColumns = {
```

Comment:
Can we have a component with user table to mutualize code for : Group.tsx, User.tsx and this one ?